.github/workflows/helm.yml

```diff
@@ -57,7 +56,6 @@ jobs:
             type=edge
             type=schedule
             type=ref,event=pr
-            type=ref,event=branch,suffix=-rc,enable=${{ startsWith(github.ref, 'refs/heads/release') }}
```

Comment:
Should these be removed either?

Reply:
See below comment - it applies to all the test image tags.